src/Site_Command.php

```diff
@@ -627,16 +913,56 @@ private function wp_download_and_config( $assoc_args ) {
 		EE::log( 'Downloading and configuring WordPress.' );
 
 		$chown_command = "docker-compose exec php chown -R www-data: /var/www/";
-		EE::debug( 'COMMAND: ' . $chown_command );
-		EE::debug( 'STDOUT: ' . shell_exec( $chown_command ) );
+		\EE\Utils\default_launch( $chown_command );
 
 		$core_download_command = "docker-compose exec --user='www-data' php wp core download --locale='" . $this->locale . "' " . $core_download_arguments;
-		EE::debug( 'COMMAND: ' . $core_download_command );
-		EE::debug( 'STDOUT: ' . shell_exec( $core_download_command ) );
+		\EE\Utils\default_launch( $core_download_command );
+
+		if ( 'db' === $this->db_host ) {
```

Comment:
Can you please add some comment here also maybe add a todo that we need to comeback to this with some proper solution if we have any with docker-compose services. ( To check mysql service is running fine in php container before executing any db operation )

Reply:
Must be added in todo. Currently the docker healthcheck for the db container can't be used due to how the image is built.

Reply:
Added